odoo/addons/base/res/res_company.py

```diff
@@ -31,7 +31,7 @@ class Company(models.Model):
         <stroke color="#000000"/>
         <lines>%s</lines>
         <!-- Set here the default font to use for all <drawString> tags -->
-        <!-- don't forget to change the 2 other occurence of <setFont> above if needed --> 
+        <!-- don't forget to change the 2 other occurence of <setFont> above if needed -->
```

Comment:
I have this onsave hook which got installed by some sublime text addons. i don't quite remember which... It's because of pep8... This is a mess, I know, but I hope it would be ok, nevertheless.